.github/workflows/publish-go-nightly-task.yml

```diff
@@ -162,9 +162,64 @@ jobs:
           name: ${{ env.ARTIFACT_NAME }}
           path: ${{ env.DIST_DIR }}/${{ env.PACKAGE_FILENAME }}
 
+  create-windows-installer:
+    runs-on: windows-latest
+    needs: create-nightly-artifacts
+
+    defaults:
+      run:
+        shell: bash
+
+    env:
+      INSTALLER_CERT_WINDOWS_PFX: "/tmp/cert.pfx"
+      SIGNTOOL_PATH: "C:/Program Files (x86)/Windows Kits/10/bin/10.0.17763.0/x86/signtool.exe"
```

Comment:
I hard-coded the path because the `signtool.exe` is not present in the windows PATH env var. There is no specific version for the version chosen, But I guess that we could use the latest available from here https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#installed-windows-sdks